docs/BR.md

```diff
@@ -1534,6 +1645,11 @@ The CA SHALL record at least the following events:
    4. Issuance of Certificates; 
    5. Generation of Certificate Revocation Lists; and 
    6. Signing of OCSP Responses (as described in [Section 4.9](#49-certificate-revocation-and-suspension) and [Section 4.10](#410-certificate-status-services)).
+   7. Multi-Perspective Domain Validation attempts from each Network Perspective, minimally recording the following information:
+      - an identifier that uniquely identifies the perspective used
+      - the attempted domain name
+      - the domain validation method used
+      - the result of the attempt (i.e., whether it corroborates both the primary CAA and domain validation determinations)
```

Comment:
It feels like this is saying two different things. I generally interpret "result" to mean "DCV pass/fail" or "CAA allow/disallow". But the parenthetical seems to define "result" as "corroborate/disagree". Which do we prefer?

Notably, if audit logs are being generated directly by systems which are performing MPDV, they probably don't _know_ whether they're corroborating or not, because at the time they get their result they don't know whether the Primary Domain Validation Determination has succeeded or not.

Reply:
The domain validation method used is captured in another existing requirement (Section 3.2.2.4: "CAs SHALL maintain a record of which domain validation method, including relevant BR version number, they used to validate every domain.").

I agree with Aaron and want to highlight that each Network Perspective acts independently of each other. The only meaningful logging from each Network Perspective would be an identifier (first bullet), the attempted domain name (second bullet), the requested type (DNS record or file downloaded) and the value obtained for each request. That would significantly increase the logging volume for CAs.

Alternatively, the CA could log the result of the corroboration based on the "Network Perspectives and Quorum Requirements" table, and capture:
- the identifiers that uniquely identify the perspectives used
- the attempted Domain Name
- the corroboration result (2 out of 3, 3 out of 3, 2 out of 4, etc.)

Reply:
It looks like this comment thread is split (not sure why). There's an additional comment from Henry [below](https://github.com/ryancdickson/staging/pull/6#issuecomment-1587502611).

Attempted to rephrase to foster continued discussion:

7. Multi-Perspective Domain Validation attempts from each Network Perspective, minimally recording the following information:
      - an identifier that uniquely identifies the perspective used
      - the attempted domain name
      - the result of the attempt (i.e., "DCV pass/fail, CAA allow/disallow")
 
8. Multi-Perspective Domain Validation quorum results for each attempted domain name represented in a Certificate request (i.e., "3/4" which should be interpreted as "3 out of 4 attempted Network Perspectives corroborated the Primary Domain Validation Determination and Primary CAA Determination).

Goals: 
- Clarify the expected result of an attempt
- Clarify that CAs should log a summary of the quorum results
- Ensure data is available to help failure analysis.